README.adoc

```diff
@@ -503,6 +503,73 @@ login_token:
 ----
 |===
 
+== Configure External Services
+
+=== Jaeger
+
+To configure the Jaeger Service you need to set the URL parameter in the config.yaml
```

Comment:
I little more wording here, it should describe which service (jaeger-query) is necessary to expose.
So I would say, "The Jaeger Service (jaeger-service) should be exposed externally, and the URL should be set in the ..." (or something like that, forgive my English here).